actions/java-gradle-release-github/README.md

```diff
@@ -4,14 +4,15 @@ This action releases Java Gradle Artifacts and a generated changelog on Github.
 
 ## Input Parameters
 
-| Name              | Required | Default Value |  Type  | Description                                                                                                   |
-| ----------------- | :------: | :-----------: | :----: | ------------------------------------------------------------------------------------------------------------- |
-| github-username   |    ✅    |       -       | string | GitHub username for requesting changes from API                                                               |
-| github-token      |    ✅    |       -       | string | GitHub token for requesting changes from API                                                                  |
-| java-distribution |    ❌    |   microsoft   | string | [Java distribution](https://github.com/actions/setup-java#supported-distributions) to be installed            |
-| java-version      |    ❌    |      11       | string | Java version to be installed                                                                                  |
-| gradle-version    |    ❌    |    wrapper    | string | [Gradle version](https://github.com/gradle/gradle-build-action#use-a-specific-gradle-version) to be installed |
-| working-directory |    ❌    |      "."      | string | Working directory of your Gradle artifacts                                                                    |
+| Name              | Required | Default Value |  Type   | Description                                                                                                   |
+| ----------------- | :------: | :-----------: | :-----: | ------------------------------------------------------------------------------------------------------------- |
+| github-username   |    ✅    |       -       | string  | GitHub username for requesting changes from API                                                               |
+| github-token      |    ✅    |       -       | string  | GitHub token for requesting changes from API                                                                  |
+| java-distribution |    ❌    |   microsoft   | string  | [Java distribution](https://github.com/actions/setup-java#supported-distributions) to be installed            |
+| java-version      |    ❌    |      11       | string  | Java version to be installed                                                                                  |
+| gradle-version    |    ❌    |    wrapper    | string  | [Gradle version](https://github.com/gradle/gradle-build-action#use-a-specific-gradle-version) to be installed |
+| working-directory |    ❌    |      "."      | string  | Working directory of your Gradle artifacts                                                                    |
+| gradle-cache      |    ❌    |     true      | boolean | Whether Gradle caching is enabled or not                                                                      |
```

Comment:
this table is sorted differently than for the other actions. the last two options are swapped. on purpose?